metadata.json

```diff
@@ -29,8 +29,8 @@
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "7",
-        "8"
+        "Stream8",
```

Comment:
For what it's worth, in Vox Pupuli we simply use `8`. This is because Facter also uses `8` as the fact `operatingsystemrelease` and CentOS Linux 8 is EOL anyway.

Reply:
You've raised a valid point. Likely was a little to hasty of me to start this work. Will halt unless I hear something from higher up on either way.

Reply:
I think it's still good to move to Stream 8 but what we did was replace the interpretation of 8 to mean stream:
* https://github.com/voxpupuli/beaker-hostgenerator/commit/a4bdd3b3c51ccc65d6e75a9fa245a2f240b0d12d
* https://github.com/voxpupuli/facterdb/issues/164 is open to do the same for FacterDB